custreamz_notebooks/streaming_word_count.ipynb

```diff
@@ -0,0 +1,275 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**This is a cuStreamz job for the classic example of Streaming Word Count.**\n",
+    "\n",
+    "For this example, we will be demonstrating how to stream from Kafka. But one can also perform a streaming word count reading from a text file which is being continuously written into."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Streamz and cudf imports\n",
+    "import cudf\n",
+    "from streamz import Stream\n",
+    "from streamz.dataframe import DataFrame"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's assume that the data coming in to the Kafka topic — i.e., each record/message, is a line in the form of \"this is line x\", where x is an incremental counter. \n",
+    "    \n",
+    "Now, we write a function to parse each such message to get the list of words in each line. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Helper function operating on every batch polled from Kafka for word count\n",
+    "def process_batch(messages):\n",
+    "    y = []\n",
+    "    for x in messages:\n",
+    "        y = y + x.decode('utf-8').strip('\\n').split(\" \")\n",
```

Comment:
May be we can shift this whole operation to `nvstrings` and rather than providing it a list of words we can directly process using `nvstrings`.  
We can use  `tokenize` here and then do a value count.  

https://rapidsai.github.io/projects/nvstrings/en/0.9.0/api.html#nvtext.tokenize

Reply:
Similar question of reading via CPU  not reading it via GPU natively via `cudf.read_csv` etc .

Reply:
Yes, I would love to give this a try. 

However, I think it'd be _okay_ to leave this as it is in this notebook, since we'd like to keep streaming word count simple, and focussed completely on the streamz functionalities.